grails-app/services/au/org/emii/gogoduck/job/JobExecutorService.groovy

```diff
@@ -1,11 +1,82 @@
 package au.org.emii.gogoduck.job
 
+import au.org.emii.gogoduck.worker.Worker
+
+import java.util.concurrent.LinkedBlockingQueue
+import java.util.concurrent.BlockingQueue
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
 class JobExecutorService {
 
+    static BlockingQueue<Map> JOB_QUEUE
```

Comment:
Do you mean `BlockingQueue` vs `LinkedBlockingQueue`?  Client code only cares that it's a `BlockingQueue` (which is an interface btw, so better to be as generic as we can be.